netmaster/master/api.go

```diff
@@ -346,7 +346,7 @@ func UpdateEndpointHandler(w http.ResponseWriter, r *http.Request, vars map[stri
 		return nil, err
 	}
 
-	if epUpdReq.Event == "start" {
+	if epUpdReq.Event == "start" || epUpdReq.Event == "connect" {
```

Comment:
Just retained the logic/ style being used in the code so far ... but yes, will change it to enumerations.